synapse/lib/stormexc.py

```diff
@@ -0,0 +1,15 @@
+class StormCtrlFlow(Exception):
```

Comment:
I'm not sure we need a whole separate file for these. These would be fine in ast.py or stormtypes.py

Reply:
The advantage to a separate file means that anyone can import without considering weird import cycles.

Reply:
For widely used or external exception i would agree, but these are internal and used in a very limited number of places.  Lets move them into `synapse/lib/ast.py`